src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
+
+interface JsonRequest {
+  method: string
+  params: unknown
```

Comment:
per your q, this is reasonable to me for now - eventually you could imagine a world where we have types that extend this one. would need to know more about the problem to opine on it though. 

my suspicion is this is either going to be some sort of interesting union type or JsonRequest is going to be a generic